src/background.ts

```diff
@@ -704,28 +704,31 @@ async function restartEngine(engineId: string) {
       runEngine(engineId);
       resolve();
     };
-    engineProcess.once("close", restartEngineOnProcessClosedCallback);
 
-    // treeKillのコールバック関数はコマンドが終了した時に呼ばれます。
-    log.info(
-      `ENGINE ${engineId}: Killing current process (PID=${engineProcess.pid})...`
-    );
-    treeKill(engineProcess.pid, (error) => {
-      // error変数の値がundefined以外であればkillコマンドが失敗したことを意味します。
-      if (error != null) {
-        log.error(`ENGINE ${engineId}: Failed to kill process`);
-        log.error(error);
-
-        // killに失敗したとき、closeイベントが発生せず、once listenerが消費されない
-        // listenerを削除してENGINEの意図しない再起動を防止
-        engineProcess.removeListener(
-          "close",
-          restartEngineOnProcessClosedCallback
-        );
-
-        reject();
-      }
-    });
+    if (engineProcess) {
```

Comment:
engineProcessがundefinedのときは、エンジンの起動に失敗した場合っぽいです。
参考↓
https://github.com/VOICEVOX/voicevox/blob/ab72c9165cedf7e1a78c7d3c490314a3341a3058/src/background.ts#L622-L626

その場合でもrestartEngine（この関数）の通りエンジンを起動しようとするのが良いのかなと思いました！
でもそれを実装していただくのはちょっとPRの内容に合致しないですね･･･↓のTODOコメントsuggestionを採用いただくだけでもOKです！

```suggestion
    // TODO: engineProcessがundefinedのときもエンジンを起動する
    if (engineProcess) {
```

Reply:
すみません、上のif文を見逃してました。。
ここでengineProcess==undefinedはあり得ない、ということがわかるように、throwにして頂ければ！